openretina/utils/plotting.py

```diff
@@ -245,6 +248,79 @@ def polar_plot_of_direction_of_motion_responses(
     plt.gca().set_xticklabels([f"{x}°" for x in sorted_directions])
 
 
+class ColorMapper:
+    def __init__(self, cmap_name: str, vmin: float = 0, vmax: float = 1):
+        self.cmap_name = cmap_name
+        self.vmin = vmin
+        self.vmax = vmax
+        self.cmap = plt.get_cmap(self.cmap_name)
+        self.norm = plt.Normalize(vmin=self.vmin, vmax=self.vmax)
+
+    def get_color(self, number):
+        color = self.cmap(self.norm(number))
+        return color
+
+
+def plot_vector_field_resp_iso(x: np.ndarray, y: np.ndarray, gradient_dict: np.ndarray,
+                               resp_dict: np.ndarray, normalize_response: bool = False,
+                               rc_dict: dict[str, Any] = {},
+                               cmap: str = "hsv") -> None:
```

Comment:
```suggestion
                               cmap: str = "hsv") -> plt.Figure:
```
This function actually returns a figure at the end